local-volume/provisioner/pkg/discovery/discovery_test.go

```diff
@@ -516,7 +516,7 @@ func TestDiscoverVolumes_NotMountPoint(t *testing.T) {
 		"dir1": {
 			{Name: "mount1", Hash: 0xaaaafef5, VolumeType: util.FakeEntryFile, Capacity: 100 * 1024},
 			// mount5 is not listed in the FakeMounter MountPoints setup for testing
-			{Name: "mount5", Hash: 0x79412c38, VolumeType: util.FakeEntryBlock, Capacity: 100 * 1024 * 1024},
+			{Name: "mount5", Hash: 0x79412c38, VolumeType: util.FakeEntryFile, Capacity: 100 * 1024 * 1024},
```

Comment:
How about adding a test for FakeEntryBlock that succeeds even through it isn't a mount point? Since this is the desired/expected behavior.

Reply:
Added a bunch of block entries named "symlink" that are not present in the fakemounter configuration. Now a bunch of tests demonstrate this scenario.